types/react-native-autocomplete-input/index.d.ts

```diff
@@ -75,15 +69,13 @@ export interface AutocompleteProps<T> extends TextInputProps {
      * function
      * render custom TextInput. All props passed to this function.
      */
-    renderTextInput?(props: TextInputProps): ReactNode;
+    renderTextInput?(props: AutocompleteProps<T>): ReactNode;
```

Comment:
As the docstring says, _all_ props are passed to `renderTextInput`, not just `TextInputProps`.